voucherify-service/src/integration/integration.service.ts

```diff
@@ -480,7 +480,7 @@ export class IntegrationService {
     });
 
     const actions = this.createActions(usedCoupons, notUsedCoupons);
-
+    console.log(response, 'RESPONSE1');
```

Comment:
:) sprawdź proszę tą PR-ke zanim dasz ją na `ready to review` możesz zawsze utworzyć drafta jeżeli wciąż nad nią pracujesz